### PR TITLE
Only you can reduce changelog clutter

### PR DESCRIPTION
**Pull request by Incoming5643**
Add some advice on how to properly keep the changelog concise and readable to a file no one ever reads any more.

### CODE REVIEW

**Comment by optimumtact** [comment]
It's only showing the last 4 weeks of changes now Incoming as I tweaked the changelog generator.

**Comment by optimumtact** [comment]
If its' still a problem I guess I could add another var for max_changelog_entries to display

**Comment by Incoming5643** [comment]
Not the problem, I'm just worried that people see a giant chain of really inconsequential entries and close the log without actually reading it. The number of entries shown doesn't actually matter here.

And yeah anyway this doesn't change anything, it's just a little catharsis.

**Comment by ChangelingRain** [comment]
:cl:
rscadd: Adds instructions on how to reduce changelog clutter.
wip: The instructions are in a file nobody reads anymore.
/:cl:

**Comment by Incoming5643** [comment]
There was a temptation

File: html/changelogs/__CHANGELOG_README.txt [hunk 1]
```diff
@@ -4,6 +4,8 @@ Changelogs are included with commits as text .yml files created individually by
 
 TO MAKE A CHANGELOG .YML ENTRRY
 
+0. Consider carefully if a change you make really needs a changelog, and if it does how many lines should be dedicated to it. A set of changes with no forward facing player effects like a refactor almost certainly doesn't need a changelog, nor does something that makes itself obvious during the normal course of play like adding an action button to an item players already commonly pick up. Likewise if you DO need a changelog consider packaging similar changes under a generalized line instead of listing out every little change as its own thing. Only you can prevent changelog clutter.
```
Comment (Jordie0608) [inline html/changelogs/__CHANGELOG_README.txt]: > nor does something that makes itself obvious during the normal course of play like adding an action button to an item players already commonly pick up.

I disagree with this.

Comment (optimumtact) [inline html/changelogs/__CHANGELOG_README.txt]: you're wrong

Comment (Jordie0608) [inline html/changelogs/__CHANGELOG_README.txt]: I'm right in my mind.

**Pull request merged**